packages/flutter/lib/src/services/message_codec.dart

```diff
@@ -187,6 +187,8 @@ class PlatformException implements Exception {
   /// ```
   ///
   /// In the cases described above, the content of [stacktrace] will be the unprocessed output of calling `toString()` on the exception.
+  ///
+  /// Platforms other than Android don't support the passing of a stacktrace to the Dart layer.
```

Comment:
It's true that macOS, linux, windows and iOS can't supply the stacktrace.  Android and Web should be able to.  Although I know we try to guide people away from platform channels on the web.  Remember also that people can implement their own platforms with the embedder API.  I'd make this more general like:

> This value will be null on platforms that don't support querying the call stack.

Reply:
Something along those lines is already the first sentence of the doc comment: https://api.flutter.dev/flutter/services/PlatformException/stacktrace.html

The background of this change is, that I was working on monitoring tools which added special handling of those native stacktraces. Finding out whether it's possible get a native stacktrace meant digging into the source code. So I wanted to make it more easy to know whether it is possible at all for other developers. 

I agree though, that it should be general for custom embedders. I haven't considered that.

Is it okay to change it to the following?
```suggestion
  /// macOS, iOS, Linux and Windows don't support querying the native stacktrace.
  ///
  /// On custom Flutter embedders this value will be null on platforms that don't support querying the call stack.
```

Reply:
Yea, that's fine to me.  It might save some users some time.